### PR TITLE
Adding content for announcing the event and opening Specialist Track CFP

### PR DESCRIPTION
**Pull request by attacus**

### CODE REVIEW

File: src/content/pages/about/index.md [hunk 1]
```diff
@@ -61,6 +55,6 @@ PyCon AU is typically held in two year blocks at the same city, with some variat
 - [2020](https://2020.pycon.org.au), [2021](https://2021.pycon.org.au): The Internet
 - (PyCon AU was not held in 2022)
 - [2023](https://2023.pycon.org.au): Tarntanya/Adelaide, SA
-- 2024: To be announced
+- 2024: Narrm/Melbourne, VIC
```
Comment (urcheraus) [inline src/content/pages/about/index.md:58]: Probably worth adding the link to the 2024 site now

File: src/content/pages/program.md [hunk 2]
```diff
@@ -4,4 +4,56 @@ title: Program
...
+In 2023, we had tracks on [Data](https://2023.pycon.org.au/data), [Education](https://2023.pycon.org.au/education), and [APIs](https://2023.pycon.org.au/ocu), as well as the perennial favourite, [DjangoCon](https://2023.pycon.org.au/djangocon).
+
+In 2024, we have space for three specialist tracks.
+
+Our specialist track organisers always do an amazing job, but we want to provide our attendees with the most awesome experience possible, and we believe that means opening the doors to potential new ideas. We want to hear from our previous specialist track organisers and others with similar experience, as well as people keen to get started in conference organising. We’d love to see submissions for tracks we’ve had in the past as well new suggestions for tracks we could feature.
+
+Examples of other ideas for specialist tracks include:
+
+- Something specialised within the data and science space, for example geoscience or life sciences
+- GLAM (Galleries, Libraries Archives and Museums)
```
Comment (urcheraus) [inline src/content/pages/program.md:24]: Missing a comma or two here

File: src/content/pages/program.md [hunk 3]
```diff
@@ -4,4 +4,56 @@ title: Program
...
+
+If you’ve got an idea that doesn’t follow the “standard” track-of-talks format, please let us know! We’d love to try new and interesting ideas. As noted above, we’ll need to work together to make sure they’re doable, and that the day runs as smoothly as possible.
+
+It is also important to note that as a specialist track organiser, you’ll be a part of the team working to make the conference happen, and one of the public faces of the conference. This means that we need to be able to trust that you’ll not only follow the code of conduct, but provide a model for the best behaviours we all wish to see in each other.
+
+## How to apply
+
+To propose a track, please email us at [program@pycon.org.au](mailto:program@pycon.org.au) before the end of **Monday, April 28th, 2024** [anywhere on earth (UTC−12)](https://en.wikipedia.org/wiki/Anywhere_on_Earth), and provide us the following information:
```
Comment (urcheraus) [inline src/content/pages/program.md:49]: Does the program@pycon.org.au address work and do we have access to it?

Comment (attacus) [reply src/content/pages/program.md:49]: It does, and we do! I have tested this.

File: src/content/pages/about/index.md (hunk 1 above)
Comment (attacus) [reply src/content/pages/about/index.md]: Good call.

File: src/content/pages/program.md (hunk 2 above)
Comment (attacus) [reply src/content/pages/program.md]: Nice catch!

**Pull request merged**